src/blocks/blocks/popup/save.js

```diff
@@ -15,7 +15,7 @@ const Save = ({
 		id: attributes.id,
 		className: classnames( className, 'is-front', { 'with-outside-button': 'outside' === attributes.closeButtonType }),
 		'data-open': attributes.trigger,
-		'data-dismiss': attributes.recurringClose ? attributes.recurringTime : '',
+		'data-dismiss': attributes.recurringClose ? attributes.recurringTime || 0 : '',
```

Comment:
This will make those blocks that are set like this to cause error in the editor. What we can do is to account for the default being zero in the frontend rather than editing the save.js method.

Reply:
@HardeepAsrani Thank you for the review! I've set the default value in frontend js, Please review my latest commit and let me know if you need any changes.

Reply:
Forgot to hit send before:

@girishpanchal30  Thanks for the update. When editing Gutenberg blocks, always keep in mind we can't change the HTML structure in the save method (save.js in our case) else the existing blocks invalidate, and we either keep it same or write a depreciation for that. If possible, we avoid editing the save method as much as posible.

Reply:
@HardeepAsrani Okay, I will keep this in mind, Thanks